numpy/random/mtrand/randint_helpers.pxi.in

```diff
@@ -23,7 +23,7 @@ def get_dispatch(dtypes):
 
 {{for npy_dt, npy_udt, np_dt in get_dispatch(dtypes)}}
 
-def _rand_{{npy_dt}}(low, high, size, rngstate):
+def _rand_{{npy_dt}}(npy_{{npy_dt}} low, npy_{{npy_dt}} high, size, rngstate):
     """
     _rand_{{npy_dt}}(low, high, size, rngstate)
 
```

Comment:
The comments below this line contradict each other. It can't be both an inclusive and closed range!

Am I right in thinking that inclusive/open is the correct description here?